parlai/tasks/saferdialogues/agents.py

```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/env python3
 
-# Copyright (c) Meta Platforms, Inc. and affiliates.
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
do we need to update our copyrights @stephenroller ?

Reply:
yes we do